docs/Using Fleet/segment-hosts.md

```diff
@@ -6,128 +6,52 @@
 ℹ️  In Fleet 4.0, Teams were introduced.
 ```
 
-- [View teams](#view-teams)
-- [Create a team](#create-a-team)
-- [Automatically adding hosts to a team](#automatically-adding-hosts-to-a-team)
+- [Overview](#overview)
+- [Naming conventions](#naming-conventions)
 - [Transfer hosts to a team](#transfer-hosts-to-a-team)
-- [Add users to a team](#add-users-to-a-team)
-- [Remove a member from a team](#remove-a-member-from-a-team)
-- [Remove a team](#remove-a-team)
+
+## Overview
 
 In Fleet, you can group hosts together in a team.
 
 With hosts segmented into exclusive teams, you can apply specific queries, policies, and agent options to each team.
```

Comment:
```suggestion
Then, you can give users access to only some teams.

This means you manage permissions so that some user can only run queries and manage hosts on the teams these users have access to.
```

Suggested edit to communicate the RBAC value of teams.

What do you think?